src/class/audio/audio_device.c

```diff
@@ -676,30 +682,30 @@ static bool audiod_decode_type_I_pcm(uint8_t rhport, audiod_interface_t* audio,
  *  Write data to buffer. If it is full, new data can be inserted once a transmit was scheduled. See audiod_tx_done_cb().
  *  If TX FIFOs are used, this function is not available in order to not let the user mess up the encoding process.
  *
- * \param[in]       itf: Index of audio function interface
+ * \param[in]       audio_fct_idx: Index of audio function interface
```

Comment:
audio_fct_idx is too long, audio prefix make it harder to read, how about `func_id`or `fid`

Reply:
I go with `func_id`!

Reply:
I would go with that as well